docs/guide/configuration.md

```diff
@@ -18,9 +18,12 @@ Main configuration files are located in the `app/config` folder.
 - `security.yml` is the place for security-related settings.
 - `routing.yml` defines routes that will be used throughout the application.
 
-Configuration can be made environment-specific using separate files for each environment. These files contain additional settings and point to the general (not environment-specific) configuration that is applied in other cases.
+Configuration can be made environment-specific using separate files for each environment.
+These files contain additional settings and point to the general (not environment-specific) configuration that is applied in other cases.
 
-Here you can read more about [how configuration is handled in Symfony](http://symfony.com/doc/current/best_practices/configuration.html).
+!!! tip
+
+    Read more about [how configuration is handled in Symfony](http://symfony.com/doc/current/best_practices/configuration.html).
```

Comment:
should we link to current or 3.4? - question to all symfony links

Reply:
We definitely should.